Magento_Checkout/templates/onepage/review/item.phtml

```diff
@@ -0,0 +1,97 @@
+<?php
+// phpcs:disable Magento2.Templates.ThisInTemplate
+
+/** @var $block Magento\Checkout\Block\Cart\Item\Renderer */
+
+$_item = $block->getItem();
+$taxDataHelper = $this->helper(Magento\Tax\Helper\Data::class);
+?>
+<tbody class="cart item">
+    <tr role="row">
+        <td
+            data-th="<?= $block->escapeHtmlAttr(__('Item:')) ?>"
+            role="gridcell"
+        >
+            <div class="row middle-sm">
+                <div class="paypal-review__product-image margin-right-m">
+                    <?= $block->getImage($block->getProductForThumbnail(), 'cart_page_product_thumbnail')->toHtml() ?>
+                </div>
+                <div class="paypal-review__product-details">
+                    <span>
```

Comment:
I think we can add some class here to have option to change styles in future projects